app/admin/css/login.scss

```diff
@@ -0,0 +1,43 @@
+@import "~bootstrap/scss/bootstrap";
+
+body {
+    padding-top: 40px;
+    padding-bottom: 40px;
+    background-color: #eee;
+}
+.form-signin {
+    max-width: 330px;
+    padding: 15px;
+    margin: 0 auto;
+}
+.form-signin #loginError {
+    display: none;
+}
+.form-signin .form-signin-heading,
+.form-signin .checkbox {
+    margin-bottom: 10px;
+}
+.form-signin .checkbox {
+    font-weight: normal;
+}
+.form-signin .form-control {
+    position: relative;
+    height: auto;
+    -webkit-box-sizing: border-box;
```

Comment:
Perhaps would it be a good idea to embed `compass-mixins` and to use them here, to prevent from worrying about prefixing.